ghost/i18n/locales/pa/portal.json

```diff
@@ -0,0 +1,216 @@
+{
+    "(save {{highestYearlyDiscount}}%)": "({{highestYearlyDiscount}}% ਬਚਾਓ)",
+    "{{amount}} days free": "{{amount}} ਦਿਨ ਮੁਫ਼ਤ",
+    "{{amount}} off": "{{amount}} ਦੀ ਛੋਟ",
+    "{{amount}} off for first {{number}} months.": "ਪਹਿਲੇ {{number}} ਮਹੀਨਿਆਂ ਲਈ {{amount}} ਦੀ ਛੋਟ।",
+    "{{amount}} off for first {{period}}.": "ਪਹਿਲੀ {{period}} ਲਈ {{amount}} ਦੀ ਛੋਟ।",
+    "{{amount}} off forever.": "ਹਮੇਸ਼ਾ ਲਈ {{amount}} ਦੀ ਛੋਟ।",
+    "{{discount}}% discount": "{{discount}}% ਛੋਟ",
+    "{{memberEmail}} will no longer receive {{newsletterName}} newsletter.": "{{memberEmail}} ਨੂੰ ਹੁਣ {{newsletterName}} ਖ਼ਬਰਨਾਮਾ ਨਹੀਂ ਮਿਲੇਗਾ।",
+    "{{memberEmail}} will no longer receive emails when someone replies to your comments.": "ਜਦੋਂ ਕੋਈ ਤੁਹਾਡੀਆਂ ਟਿੱਪਣੀਆਂ ਦਾ ਜਵਾਬ ਦੇਵੇਗਾ ਤਾਂ {{memberEmail}} ਨੂੰ ਹੁਣ ਈਮੇਲ ਪ੍ਰਾਪਤ ਨਹੀਂ ਹੋਵੇਗੀ।",
```

Comment:
AI has flagged this, and suggests ਹੁਣ ਜਦੋਂ ਕੋਈ ਤੁਹਾਡੀਆਂ ਟਿੱਪਣੀਆਂ ਦਾ ਜਵਾਬ ਦੇਵੇਗਾ, {{memberEmail}} ਨੂੰ ਈਮੇਲ ਨਹੀਂ ਭੇਜੀ ਜਾਵੇਗੀ। instead.